package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@smartbear/mcp",
-  "version": "1.0.0",
+  "version": "0.1.0",
```

Comment:
if you can run `npm install` to update the lock file, that would be good

Reply:
ahhhh sorry - I synced too quickly. Should be good now